docs/topics/command-line.md

```diff
@@ -133,9 +137,10 @@ kotlinc -script list_folders.kts -- -d <path_to_folder_to_inspect>
 ```
 
 Kotlin provides experimental support for script customization, such as adding external properties,
-providing static or dynamic dependencies, and so on. Customizations are defined by so-called *Script definitions* -
-annotated kotlin classes with the appropriate support code. The script filename extension is used to select the appropriate
-definition. Learn more about [Kotlin custom scripting](custom-script-deps-tutorial.md).
+providing static or dynamic dependencies, and so on.
+Customizations are defined by so-called _Script definitions_ – annotated kotlin classes with the appropriate support code.
```

Comment:
Should "Script definitions" be capitalized here? On the page linked it looks like a simple term.

Reply:
Updated!